cisco_sdwan/metadata.csv

```diff
@@ -0,0 +1,32 @@
+metric_name,metric_type,interval,unit_name,per_unit_name,description,orientation,integration,short_name,curated_metric
+cisco_sdwan.bfd_session.status,gauge,,,,The status of the BFD session.,0,cisco_sdwan,,
+cisco_sdwan.control_connection.status,gauge,,,,The status of the control connection.,0,cisco_sdwan,,
```

Comment:
non blocking nit. Not sure the values of these metrics, but if they're status metrics like 1 for up 0 for down, would it make sense to document these in the description?

Reply:
Good point thanks, updated in c31a3b4